blivet/devicefactory.py

```diff
@@ -348,6 +350,39 @@ def __init__(self, storage, **kwargs):
         self.__names = []
         self.__roots = []
 
+    def _update_defaults_from_device(self):
+        """ Set up default settings based on device, if provided, or internal defaults. """
```

Comment:
I find the docstring a bit misleading as the method doesn't seem to update the default settings based on internal defaults. Maybe just drop the *"or internal defaults." part?